src/main/java/com/textmessenger/model/entity/dto/Token.java

```diff
@@ -0,0 +1,13 @@
+package com.textmessenger.model.entity.dto;
+
+import lombok.Data;
+
+@Data
+public class Token {
+  private String accessToken;
+  private static final String TOKEN_TYPE = "Bearer";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "TOKEN_TYPE" private field. ([squid:S1068](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=142&issues=AWWFtK2YXKENQn1S-W5d&open=AWWFtK2YXKENQn1S-W5d)

[ISSUE_KEY]: AWWFtK2YXKENQn1S-W5d